docs/content/latest/integrations/flyway.md

```diff
@@ -0,0 +1,118 @@
+
+  ```properties
+  flyway.url=jdbc:postgresql://localhost:5433/yugabyte
+  flyway.user=yugabyte
+  flyway.password=yugabyte
+  ```
+
+## Migrating schema
+
+Flyway allows you to specify migrations using either SQL or Java.
+
+{{< note title="Note" >}}
+
+By default, Flyway runs migrations inside a transaction. In case of failures, the transaction is rolled back (see [Flyway Transactions](https://flywaydb.org/documentation/concepts/migrations.html#transactions)). Since YugabyteDB does not currently support DDLs inside a user-initiated transaction (instead, it runs a DDL inside an implicit transaction), you may need to manually revert the DDL changes when you see a message about failed migrations “Please restore backups and roll back database and code”.
+
+{{< /note >}}
+
+### How to use SQL
+
+You can specify migrations as SQL statements in `.sql` files that are placed in the `<FLYWAY_INSTALL_DIR>/sql/` directory by default. You can change the location of these files by placing them in a different directory and then editing the `flyway.locations` property in the `flyway.conf` file accordingly.
+
+To migrate schema using SQL, perform the following:
+
+- Create migration SQL scripts under the `sql/` directory, as shown in the following example using the `motorcycle_manufacturers` table:
+
+  ```plsql
+  cat sql/Create_motorcycle_manufacturers_table.sql
+  ```
+
+  ```sql
+  CREATE TABLE motorcycle_manufacturers (
+    manufacturer_id SERIAL PRIMARY KEY,
+    manufacturer_name VARCHAR(50) NOT NULL
+  );
+  ```
+
+  ```plsql
+  cat V2__Insert_into_motorcycle_manufacturers.sql
+  ```
+
+  ```sql
+  INSERT INTO motorcycle_manufacturers (manufacturer_id, manufacturer_name) 
+  VALUES (default, 'Harley-Davidson'), (default, 'Yamaha');
+  ```
+
+- Run migrations by executing the following command:
+
+  ```shell
+   ./flyway migrate
+  ```
+
+### How to use Java
+
+You can define Flyway migrations as Java classes by extending the `BaseJavaMigration` class and overriding the `migrate()` method.
+
+The following example shows how to use Java to add a column called `hq_address` to the `motorcycle_manufacturers` table:
+
+```java
+package db.migration;
+
+import org.flywaydb.core.api.migration.BaseJavaMigration;
+import org.flywaydb.core.api.migration.Context;
+import java.sql.Statement;
+
+public class V3__AddHQAdress extends BaseJavaMigration {
+
+  public void migrate(Context context) throws Exception {
+    try (Statement alter = context.getConnection().createStatement()) {
+      alter.execute("ALTER TABLE motorcycle_manufacturers 
+                    ADD COLUMN hq_address VARCHAR(50)");
+    }                   
+  }
+}
+```
+
+To check the state of the database and run the migration, execute the following commands:
+
+```shell
+ ./flyway info
+```
+
+```shell
+ ./flyway migrate
+```
+
+
```

Comment:
Can we add a link to flyway documentation (https://flywaydb.org/documentation/getstarted/), may be at the end?

Reply:
Added a link to Flyway at the top